### DISCUSSION TITLE
Build and install mpd - default mpd.conf missing

### DISCUSSION THREAD

**Discussion by slippers** [MusicPlayerDaemon/MPD#1597] [General]
using mpd-0.23.9 tarball source files to build and install mpd.

Was expecting a default mpd.conf file deployed as part of the install.

Is this file doc/mpdconf.example the default?

**Comment by MaxKellermann** [comment]
There is no default `mpd.conf`. Actually, `mpd.conf` is optional. Create it only if you need different settings than the defaults, and add only those options, nothing else.
